OsmIntegrator/Database/Models/JsonFields/ReportField.cs

```diff
@@ -10,10 +12,14 @@ public class ReportField
     public string PreviousValue { get; set; }
     public ChangeAction Action { get; set; }
 
-    public override string ToString()
+    private string GetResultText(IStringLocalizer<TileController> localizer = null)
```

Comment:
Sugestia
Nie ma sensu sprawdzania, czy `localizer` jest nullem ponieważ jest przekazywany z obiektu nadrzędnego (kontrolera). Jeżeli nie byłby zainicjowany przez Dependency Injection to IoC zgłosiłby błąd, że brakuje serwisu.
Już lepiej jest rzucić wyjątek podczas parsowania argumentów np. 
```if(localizer == null) throw new ArgumentNullException("Localizer shouldn't be null");```
ale jest to wg. mnie opcjonalne.

Reply:
Racja, wywaliłem wszystkie sprawdzenia czy jest nullem